### ISSUE TITLE
Glimpse not detecting ajax request done in $(document).ready()

### ISSUE THREAD

**Issue by jmazurpl** [Glimpse/Glimpse#397]
Glimpse is not detecting the ajax requests made in $(document).ready(). If I trigger the same ajax request for example with button everything is detected as it should.

**Comment by avanderhoorn** [comment]
@NickLarsen Agreed with this. Glimpse v2 will do things differently on this front. Hence making sure we can capture everything that happens.

**Comment by avanderhoorn** [comment]
Note, if someone wants to make the change for the v1 client, I'm happy to help with the effort.

**Comment by mpetito** [comment]
@avanderhoorn, I've been tinkering with this issue for v1.  My thoughts are to attach the ajax listener in a separate script loaded somewhere in the document head.  The listener will reference the pubsub module and a stub will be included to queue up requests until the glimpse client is loaded.  Once the glimpse client is loaded, the stub pubsub is replaced and queued messages are delivered as the corresponding subscriptions are registered.  I've made these changes to the client (including a new glimpse.head.js file) here: https://github.com/mpetito/Glimpse.Client/commit/2ba16d2502c463a1381c1091f6b4563574da11cf 

I've tested these changes by modifying the glimpse.js resource in `Glimpse.Core` and manually adding glimpse.head.js to the document head.

One potential issue with this approach is that ajax requests initiated after the pubsub module loads, but before the hud module's `postRender` method subscribes to `ajax.update`, will not be observed.

I'm also not sure how best to include the new glimpse.head.js file in the document head.  `Glimpse.Core.Framework.IFrameworkProvider.InjectHttpResponseBody` expects a single `htmlSnippet` for inclusion using `Glimpse.AspNet.PreBodyTagFilter`.  It seems that, in addition to the embedded script resource classes in `Glimpse.Core`, `IFrameworkProvider` needs another method for `InjectHttpResponseHead`?
